### PR TITLE
feat: support either defineStore(id, options) or defineStore(options)

### LINKED ISSUE #16: Stores registered with feathers-pinia not showing-up in the devtools
**TLDR:** Original `defineStore()` called with `options.storeId` instead of `options.id` and pinia requires `options.id` to register store in vue devtools.

### Issue

Stores created with feathers-pinia not showing-up in vue devtools (at least when using vue2).

### Context

From pinia docs: 
> This name, also referred as id, is necessary and is used by Pinia to connect the store to the devtools.

### Solution

In this case, the only fix should be to rename the expected `options.storeId` to `options.id` to at least have a more transparent/direct mapping to the pinia API.  This change does not break tests and remains backwards compatible.

So basically change the following code:
```diff
  // Create and initialize the Pinia store.
  const storeOptions: any = makeServiceStore({
-   storeId: options.id || `service.${options.servicePath}`,
+   id: options.id || `service.${options.servicePath}`,
    idField: options.idField || idField || 'id',
    clientAlias,
    servicePath,
    clients,
    Model: options.Model,
    state,
    getters,
    actions,
    whitelist,
  })
  function useStore(pinia?: Pinia): any {
    const useStoreDefinition = definePiniaStore(storeOptions)
    ...
  }
```

### PR DESCRIPTION
**Pull request by miguelrk**
This PR adds support for passing either two arguments `defineStore(id, options)` or a single one `defineStore(options)` to better align this with pinia. Tests are updated and passing to cover for 4 cases:
1. `defineStore({ servicePath: 'users', ... })`
2. `defineStore({ id: 'users2', servicePath: 'users', ... })`
3. `defineStore('users3', { servicePath: 'users', ... })`
4. `defineStore('users4', { id: 'should-be-overriden', servicePath: 'users', ... })`

Closes #16

### CODE REVIEW

**Comment by miguelrk** [comment]
@marshallswain what's your take on 
```ts
options.id = id || `service.${options.servicePath}`
```
should we default to `service.${options.servicePath}` or more simply to `options.servicePath` ? If we keep `options.servicePath` we would only need to update some tests, else this is ready 👍🏼

File: src/setup.ts [hunk 1]
```diff
@@ -23,7 +23,10 @@ export function setupFeathersPinia(globalOptions: SetupOptions) {
     registerClient(name, clients[name])
   })
 
-  function defineStoreWrapper(options: DefineStoreOptions) {
+  function defineStoreWrapper(...args: [DefineStoreOptions] | [string, DefineStoreOptions]) {
```
Comment (marshallswain) [inline src/setup.ts:26]: This is really cool syntax that I haven't seen before.  TIL 🙏

**Comment by marshallswain** [comment]
I prefer that the default has a prefix.  On large projects it's too easy to lose track of service stores.  Thanks for the PR.

**Comment by marshallswain** [comment]
Released as `feathers-pinia@0.21.0`. :pray:

**Comment by marshallswain** [comment]
I didn't even think about docs.  Do you recall any examples that might need updating.  I can look into it later if you don't have time for it.

**Comment by miguelrk** [comment]
Oh haha me neither. Count on it, no worries 👍🏼

**Comment by miguelrk** [comment]
Just realized the `defineAuthStore` is also missing support for both signatures, I'm on it right now and will open up another PR soon.

**Pull request merged**